### PR TITLE
Michelson editor and Local nodes integration to the GUI.

### PR DESCRIPTION
**Pull request by aditya-tezsure**
This Fix includes the following changes.
- Adding Blocks explorer.

- Refactoring accounts section.
 
- Adding Michelson editor to the GUI.

- Integrating Localnodes to the GUI.

- clickable area to be shown with hand cursor.

- Input any receiver's address to send transactions.

- Making a password non-mandatory field.

- Removing CLI not found startup message.

- And various small UI-fixes.

### CODE REVIEW

**Comment by jvtezsure** [comment]
@aditya-tezsure pls provide proper title to PR.

File: app/actions/Onboard.js [hunk 1]
```diff
@@ -16,53 +18,42 @@ export function checkTezsterCliAction() {
...
+    RpcRequest.checkNodeStatus(url)
+      .then((res) => {
+        if (res.protocol.startsWith('PsCARTHAG')) {
+          return setTimeout(() => {
+            dispatch({
+              type: 'TEZSTER_SHOW_STOP_NODES',
+              payload: true,
+            });
+            return dispatch({
+              type: 'TEZSTER_CLI_SUCCESS',
```
Comment (jvtezsure) [inline app/actions/Onboard.js:30]: remove reference to CLI. You're not downloading or setting up CLI.
It'll create unnecessary confusion in the codebase.

**Comment by jvtezsure** [comment]
It's Michelson.

File: app/actions/Onboard.js [hunk 2]
```diff
@@ -72,3 +62,56 @@ export function getLocalConfigAction() {
...
+          });
+          return dispatch({
+            type: 'TEZSTER_NODES_ERR',
+            payload: false,
+          });
+        }, 1000);
+      });
+  };
+}
+export function getTezsterCliRunningState() {
```
Comment (jvtezsure) [inline app/actions/Onboard.js:104]: remove all CLI references please.

**Comment by jvtezsure** [comment]
Title mentiones CLI integation. Please update that as well.

File: app/actions/Onboard.js (hunk 2 above)
Comment (aditya-tezsure) [reply app/actions/Onboard.js]: done

**Pull request merged**